Vagrantfile

```diff
@@ -56,7 +56,7 @@ Vagrant.configure(2) do |config|
     config.vm.provision "shell", inline: "apt-get update"
 
     # Install build dependencies
-    config.vm.provision "shell", path: "cookbook/system-config/setup-ccache-and-icecc.sh"
+#    config.vm.provision "shell", path: "cookbook/system-config/setup-ccache-and-icecc.sh"
```

Comment:
Alright, might be best to add a comment, since there is a reason, but perhaps a bit too cumbersome to make an update only for this.